.github/workflows/database-migrations.yml

```
@@ -0,0 +1,159 @@
+  MYSQL_SERVER_NAME: web-wizards-mysql
+  MYSQL_DATABASE_NAME: webwizardsdb
+  ENVIRONMENT: production
+  AZURE_REGION: southeastasia  # Updated for Bangladesh users
+
+jobs:
+  setup-database:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)